source/common/tls/connection_info_impl_base.h

```diff
@@ -88,6 +90,8 @@ class ConnectionInfoImplBase : public Ssl::ConnectionInfo {
     IpSansPeerCertificate,
     OidsPeerCertificate,
     OidsLocalCertificate,
```

Comment:
Note to self: we could probably drop this in favor of the `OidMapPeerCertificate`/`OidMapLocalCertificate` implementation, and just iterate over keys when needed.